alacritty_terminal/src/tty/windows/conpty.rs

```diff
@@ -241,8 +241,9 @@ pub fn new(config: &Options, window_size: WindowSize) -> Option<Pty> {
     Some(Pty::new(conpty, conout, conin, child_watcher))
 }
 
-// https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-createprocessa#parameters
-// https://learn.microsoft.com/sk-SK/previous-versions/troubleshoot/windows/win32/createprocess-cannot-eliminate-duplicate-variables#environment-variables
+// Windows environment variables are case-insensitive, and
+// https://learn.microsoft.com/en-us/previous-versions/troubleshoot/windows/win32/createprocess-cannot-eliminate-duplicate-variables#environment-variables
+// mentions that the caller is responsible for deduplicating environment variables, so do that here while converting.
```

Comment:
Docs should be written either using markdown syntax with putting long links beyond the URL, or since these comments are never rendered in markdown they should be written in plain text while adding references below, the way you did it with your other comment.

In its current state this is unreadable.

Reply:
Ok, moved the URL in the very bottom of the comment.